src/node_os.cc

```diff
@@ -290,7 +291,7 @@ static void GetInterfaceAddresses(const FunctionCallbackInfo<Value>& args) {
            env->family_string(), family).FromJust();
     o->Set(env->context(),
            env->mac_string(),
-           FIXED_ONE_BYTE_STRING(env->isolate(), mac)).FromJust();
+           OneByteString(env->isolate(), mac.data(), mac.size())).FromJust();
```

Comment:
So I think here you need to pass the length of the string not the length of the buffer.
I see several solutions:
1. use `mac.size() - 1` like `FIXED_ONE_BYTE_STRING`  does:
https://github.com/nodejs/node/blob/4270c13426fb8d86f1068735006b95bb614e9f38/src/util.h#L218-L224
2. a less performant solution would be to use `std::strlen(mac.data())`;
3. Add an overload of FIXED_ONE_BYTE_STRING that takes `std::array`:
```cpp
 // Used to be a macro, hence the uppercase name. 
 template<std::size_t SIZE>
 constexpr v8::Local<v8::String> ToFixedOneByteString( 
     v8::Isolate* isolate, 
     std::array<char, SIZE> arr) { 
   return OneByteString(isolate, arr.data(), SIZE - 1); 
 }
```

Reply:
Solution 1 should work; solution 2 is equivalent to omitting the last parameter.